package.json

```diff
@@ -20,12 +20,14 @@
     "@ts-tools/node": "^0.8.3",
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.5",
+    "@types/node": "^20.14.10",
     "chai": "^4.2.0",
     "mocha": "^5.2.0",
     "tslint": "^5.12.0",
     "typescript": "^4.2.4"
   },
   "dependencies": {
+    "dotenv": "^16.4.5",
```

Comment:
@galvin96 , i wonder why we need `dotenv` here?

Reply:
In this file: [test/amoy.test.ts](https://github.com/IX-Swap/ethers-multicall/pull/1/files/00715a11f46bc54349397057cfe92ae06669d808#diff-8649aaa272cb5cbd432f8b2c9c4fb62168338288bb5eaa6b3253007ccf137c59) we need to retrieve the RPC url from the .env file. Thus, I added this package.

Reply:
Went with the 2nd approach. updated.